test/e2e/api_test.go

```diff
@@ -59,6 +73,11 @@ func TestRegisterEvents(t *testing.T) {
 				Event: goalEvent,
 				Type:  api.GoalEventType,
 			},
+			{
```

Comment:
Please rebase this branch and use the `newEvent` that was introduced here.

https://github.com/ca-dp/bucketeer-go-server-sdk/pull/91/files#diff-40be257af42e6fcbb9b3cb9b1e41df34ac9d1b83ee2c32a91a4722389b196060R331

Reply:
I don't want to make `newEvent` func public for tests.